### PR TITLE
Force pushing has its use

### PR DESCRIPTION
**Pull request by readefries**
As you can keep the history clean and sane. Making merges easier.

### Why:

Force pushing has its use and can add value to prevent pollution in your commit history. I would not advocate against using it, only towards using it wisely ;)

### Check off the following:

- [x] I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by cloudcosmonaut** [comment]
Force pushing typo's, as they don't add any value in the commit history ;)

**Comment by ramyaparimi** [comment]
@cloudcosmonaut Thanks so much for opening a PR! I'll get this triaged for review :zap:

File: content/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ topics:
 
 **Note:** When working with pull requests, keep the following in mind:
 * If you're working in the [shared repository model](/pull-requests/collaborating-with-pull-requests/getting-started/about-collaborative-development-models), we recommend that you use a topic branch for your pull request. While you can send pull requests from any branch or commit, with a topic branch you can push follow-up commits if you need to update your proposed changes.
-* When pushing commits to a pull request, don't force push. Force pushing changes the repository history and can corrupt your pull request. If other collaborators branch the project before a force push, the force push may overwrite commits that collaborators based their work on.
+* Be very careful when using force pushing changes to a pull request. Force pushing can corrupt your pull request. Rabase your pull request branch based on the default branch. By doing this, you keep the history clear and sane. If other collaborators branch the project before a force push, the force push may overwrite commits that collaborators based their work on.
```
Comment (lecoursen) [inline content/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests.md:24]: I agree that we don't have to prohibit force pushing entirely, but we also want the article to serve a beginner audience, who may be confused by mention of rebasing. Let's soften the first sentence and keep the rest as is.

```suggestion
* Be very careful when force pushing commits to a pull request. Force pushing changes the repository history and can corrupt your pull request. If other collaborators branch the project before a force push, the force push may overwrite commits that collaborators based their work on.
```

**Review by lecoursen — DISMISSED** [review]
Thanks @cloudcosmonaut! I'm going to apply a suggestion and then merge. ⚡

**Comment by cloudcosmonaut** [comment]
Something is going wrong here, but merging in the changes. I can rebase `main` into this branch, so everything is up to date. As the merging removed my changes :/

**Comment by lecoursen** [comment]
>Something is going wrong here, but merging in the changes. I can rebase main into this branch, so everything is up to date. As the merging removed my changes :/

@cloudcosmonaut, I intentionally updated the branch in https://github.com/github/docs/commit/166f9c61d23c785a8b0ecde637d27d716619bcd5. It looks like your force pushing has overwritten my changes, much like the note in question suggests. 🙃 

We cannot accept the change as is, so I am going to reapply my suggestion and then merge.

File: content/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests.md (hunk 1 above)
Comment (lecoursen) [inline content/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests.md:24]:
```suggestion
* Be very careful when force pushing commits to a pull request. Force pushing changes the repository history and can corrupt your pull request. If other collaborators branch the project before a force push, the force push may overwrite commits that collaborators based their work on.
```

**Review by lecoursen — APPROVED** [review]

**Comment by cloudcosmonaut** [comment]
But why can't you add to part of rebasing? I think it's very important to keep your history sane

**Comment by lecoursen** [comment]
> But why can't you add to part of rebasing? I think it's very important to keep your history sane

I explained that in my review comment (https://github.com/github/docs/pull/15286#discussion_r802989246):

> I agree that we don't have to prohibit force pushing entirely, but we also want the article to serve a beginner audience, who may be confused by mention of rebasing. Let's soften the first sentence and keep the rest as is.

**Comment by cloudcosmonaut** [comment]
Sorry I missed that part. Would you be open for a section later in the doc?

**Pull request merged**